cmd/virt-launcher/virt-launcher.go

```diff
@@ -229,6 +229,7 @@ func main() {
 	readinessFile := flag.String("readiness-file", "/tmp/health", "Pod looks for this file to determine when virt-launcher is initialized")
 	gracePeriodSeconds := flag.Int("grace-period-seconds", 30, "Grace period to observe before sending SIGTERM to vm process")
 	allowEmulation := flag.Bool("allow-emulation", false, "Allow fallback to emulation if /dev/kvm is not present")
+	requestedHooks := flag.Uint("requested-hooks", 0, "Number of requested hooks, virt-launcher will wait for all of them to become available")
```

Comment:
Requested hooks at first - I thought was a filter on the endpoints, but what this is - is a `sidecars-hooks` - minor nitpick

Reply:
yeah, lgtm